packages/appeals-service-api/package.json

```diff
@@ -7,7 +7,14 @@
 		"npm": ">=9.0.0 <10.0.0"
 	},
 	"main": "src/main.js",
+	"prisma": {
+		"schema": "src/db/prisma/schema.prisma",
+		"seed": "node src/db/prisma/seed/seed.js"
+	},
 	"scripts": {
+		"db:migrate:dev": "npx prisma migrate dev",
+		"db:generate": "npx prisma generate",
+		"db:seed": "npx prisma db seed",
 		"lint": "eslint ./",
 		"lint:fix": "eslint ./ --fix",
 		"start": "node ./",
```

Comment:
We need to run generate before  `start:dev` else we won't have a generated client to use. Just `npm run db:generate && ...` should do it.

We probably need to add `npm run db:generate` to the Dockerfile for the API to do this at build time as well.

Reply:
Or can we just add `npm run db:generate` before the `start` script since this is what Docker calls?

Reply:
Sure, either way works!